packages/react/src/hooks/__tests__/useTheme.test.tsx

```diff
@@ -0,0 +1,41 @@
+import { createTheme, defaultTheme, WebTheme } from '@aws-amplify/ui';
+import { renderHook } from '@testing-library/react-hooks';
+import { AmplifyProvider } from '../../components/AmplifyProvider';
+import { useTheme } from '../useTheme';
+
+const serializeTheme = (theme: WebTheme) => JSON.stringify(theme);
```

Comment:
have to serialize themes before comparing them to test equality: https://github.com/facebook/jest/issues/8475